### DISCUSSION TITLE
How to setup presto on aarch64? (Presto requires amd64 or ppc64le on Linux (found aarch64))

### DISCUSSION THREAD

**Discussion by zhihuawei** [prestodb/presto#18674] [Q&A]
I used ubuntu 20.4 server build a simple cluster on my mac (m1 pro) and setup presto in the cluster。
when i start presto-server console
<img width="790" alt="image" src="https://user-images.githubusercontent.com/21271637/201516633-7f2bc0ce-dc2f-49a5-b86a-b08381216bb5.png">

```
Presto requires amd64 or ppc64le on Linux (found aarch64)
```

How can I solve this problem, can anybody help me ?

**Comment by KIRITOLTR** [comment]
version 0.286 supports aarch64,mayby you can try it out.
